contents/blog/brand.md

```diff
@@ -0,0 +1,128 @@
+---
+date: 2023-06-25
+title: How to brand your startup
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+  - james-hawkins
+category: CEO diaries
+---
+
+The world would be a lot more fun if most startups didn't seem to have experienced a personality vasectomy.
+
+I believe humans have evolved to conform - when we all lived in the wilderness, staying alive required you to fit in with a tribe, after all. Fast forward that background to today and, amongst other things (!), the majority of the world's software companies feel the same as each other.
+
+## There are good reasons for this, you idiot
```

Comment:
the 'voice' of the crossheads changes. here it's the 'dissenting voice' but lower down the crossheads just become crossheads. think we need to pick one and stick with it